.changeset/lucky-apples-serve.md

```diff
@@ -0,0 +1,6 @@
+---
+'@lit-labs/task': minor
+---
+
+Added `initialValue` parameter to `TaskConfig`.
+Task can start with an initial value from TaskConfig.
```

Comment:
We are changing the behavior of how value is treated when the task is re-run. This should be called out specifically here.